server/routes/incidentDetails/incidentDetailsValidation.ts

```diff
@@ -80,11 +81,15 @@ export default function validateForm({
   if (incidentDate.time.minute.length < 2) {
     return errors.ONE_DIGIT_MINUTE
   }
-  if (incidentDate.date === `${now.getDate()}/${now.getMonth() + 1}/${now.getFullYear()}`) {
-    if (Number(incidentDate.time.hour) > now.getHours()) {
+
+  if (incidentDate.date === getDate(now, 'DD/MM/YYYY')) {
```

Comment:
Surely you don't need to strip the hours and minutes?!

Can'y you just get the UTC timestamp (`new Date().getTime()`) and diff that with the provided time?

It will require converting the provided hours and minutes to a UTC, but that should be pretty easy.

Worth investigating...

Reply:
Yeah you're right, must have been 6pm delirium